codes/go/chapter_stack_and_queue/stack_test.go

```diff
@@ -0,0 +1,89 @@
+// File: stack_test.go
+// Created Time: 2022-11-28
+// Author: Reanon (793584285@qq.com)
+
+package chapter_stack_and_queue
+
+import "testing"
+
```

Comment:
Testing for the Go built-in stack is required.

Reply:
In actual, there is't a built-in stack in Go. So, i try to define a Go Interface to be aligned with Java code.